### PR TITLE
Fix UIWindow size for iOS9

### PR DESCRIPTION
**Pull request by jontelang**
Issue: iPads on iOS9 sometimes have the window not big enough. Especially when INVOKING the DimWindow WHILE in landscape mode.

How I tested this (I don't own an iPad on iOS9) I created an Xcode project, included the DimWindow and DimController and simply called the [DimController sharedInstance] which adds the window. I had to set some preferences values and call to update them after the window was created (5s delay).

So first start an iPad simulator in portrait mode, should work fine I think (cant actually remember). Then put it in landscape mode and restart the project. It should now look like this:

image

Basically the coordinate system is messed up.

Now after the code in this PR it just works, like this: 

image

### CODE REVIEW

**Comment by thomasfinch** [comment]
Thanks! I replicated this on my iPad following your instructions and the fix worked for me.

**Comment by jontelang** [comment]
Nice :+1:
